fastlane/lib/fastlane/actions/docs/frame_screenshots.md

```diff
@@ -70,16 +83,69 @@ To download the latest frames
 
 When using _frameit_ without titles on top, the screenshots will have the full resolution, which means they can't be uploaded to the App Store directly. They are supposed to be used for websites, print media and emails. Check out the section below to use the screenshots for the App Store.
 
-# Titles and Background (optional)
+## Advanced Usage (optional)
+
+### Text and Background
 
-With _frameit_ 2.0 you are now able to add a custom background, title and text colors to your screenshots.
+With _frameit_ it's possible to add a custom background and text below or above the framed screenshots in fonts and colors you define.
 
 A working example can be found in the [fastlane examples](https://github.com/fastlane/examples/tree/master/MindNode/screenshots) project.
 
-#### `Framefile.json`
+### `Framefile.json`
+
+The Framefile allows to define general and screenshot specific information.
+It has the following JSON structure:
+
+```json
+{
+  "device_frame_version": "latest",
+  "default": {
+    ...
+  },
+  "data": [
+     ...
+  ]
+}
+```
+
+### General parameters
 
-Use it to define the general information:
+The general parameters are defined in the `default` key and can be:
 
+| Key | Description | Default value |
```

Comment:
This should not be here. The _frameit_'s source code should be changed (create `ConfigItem`s for these missing options) so the docs would be correctly generated.

Reply:
see my comment below on the main PR comments thread

Reply:
I see now, didn't know the difference between _frameit_ and the rest of the tools. Knowing this, you should disregard this review comment, at least for now.